### DISCUSSION TITLE
Cast Segmentation fault

### DISCUSSION THREAD

**Discussion by kingmanas** [Cyfrin/foundry-full-course-cu#1574] [Q&A]
Cast is suddenly giving Segmantation Fault previously it was working fine..can someone give the solution for it

**Comment by EngrPips** [comment]
Hello @kingmanas , please run `foundryUp` and then retry using cast.

**Reply by kingmanas** [reply]
root@BlockBuddy:~/foundry-f24/foundry-nft-f24# foundryUp
foundryUp: command not found

not Found..!!!!

**Reply by EngrPips** [reply]
are you using `wsl` ?

**Reply by kingmanas** [reply]
yes

**Reply by EngrPips** [reply]
Honestly never used `wsl` my whole life so it is kinda difficult for me to even know what is going on but i would ask with people that knows about `wsl` and get back to you with whatever i find.
